lib/sandbox/pmapi.js

```diff
@@ -398,6 +407,15 @@ function Postman (execution, onRequest, onSkipRequest, onAssertion, cookieStore,
             }
         });
     }
+    // add message assertions
+    if (this.message) {
```

Comment:
maybe (need to think more): we can leave this part out as well and offload it to template.